src/utils/decode_url_params.js

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
would it make sense to put both the encode and decode methods in a single file, maybe called url_params.js ?